src/components/CountryCard.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.